Azure/azure-cli
# [AKS] `az aks update`: Add forceupgrade settings to aks stable cli

**Related command**
add forceupgrade settings to aks stable cli

**Description**


**Testing Guide**
aks update --resource-group={resource_group} --name={name} --enable-force-upgrade

This checklist is used to make sure that common guidelines for a pull request are followed.
